lib/jobs/ServiceInstanceUpdateJob.js

```diff
@@ -198,10 +198,11 @@ class ServiceInstanceUpdateJob extends BaseJob {
           //Fetch last 3 run history records and figure out the status of the update operation.
         })
       .then(lastRunDetails => {
-        logger.info('LastRun details retrieved - ', lastRunDetails);
+        logger.info('Total last runs # ', _.get(lastRunDetails, 'totalRecordCount', 0));
```

Comment:
Specification of last run details of job SI update would be better.

Reply:
This was causing problems earlier. `lastRunDetails` array is mongo db objects of lastrundetails & they are huge. They not only contain the data representation but it also prints various other attribs of mongo wrapper classes, which are really not required. What really interests us is the first run response and last run response which are logged appropriately.

Reply:
I meant in logger the job name: ServiceInstanceUpdateJob and instance id would be better. As it will keep a reference the id against the count. No other details.